### PR TITLE
feat(FOROME-902): redesign header, add component for showing count of…

### PR DESCRIPTION
**Pull request by QSdmitrioul**
… variants

https://quantori.atlassian.net/browse/FOROME-902

### CODE REVIEW

File: src/ui/button.tsx [hunk 1]
```diff
@@ -85,7 +87,7 @@ export const Button = ({
       'hover:border-blue-hover': isSecondary || isSecondaryDark,
       'active:border-blue-active': isSecondary || isSecondaryDark,
       //default diestruction
-      'border-2 border-red-secondary': isDiestruction,
+      'border-2 border-red-secondxary': isDiestruction,
```
Comment (Lapk1n) [inline src/ui/button.tsx]: Excess letter

File: src/ui/button.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { CSSProperties } from 'styled-components'
 
 export interface ButtonProps {
   text?: string | JSX.Element
+  textSize?: 'xs' | 'sm' | 'md'
```
Comment (Lapk1n) [inline src/ui/button.tsx]: The max size of the button's text is 14px

File: src/ui/button.tsx (hunk 1 above)
Comment (QSdmitrioul) [reply src/ui/button.tsx]: fixed it

**Review by AndreyUstyumenko — APPROVED** [review]

File: src/ui/button.tsx (hunk 2 above)
Comment (QSdmitrioul) [reply src/ui/button.tsx]: Fixed

**Review by Lapk1n — APPROVED** [review]

**Pull request merged**